docs/docs/cmd/spo/homesite/homesite-set.md

```diff
@@ -47,6 +63,28 @@ m365 spo homesite set --siteUrl https://contoso.sharepoint.com/sites/comms
     ```csv
     The Home site has been set to https://contoso.sharepoint.com. It may take some time for the change to apply. Check aka.ms/homesites for details.
     ```
+### `vivaConnectionsDefaultStart` response
+
+When we make use of the option `vivaConnectionsDefaultStart` the response will differ. 
+
+=== "JSON"
+
+    ```json
+    "The Home site has been set to https://contoso.sharepoint.com and the Viva Connections default experience to True. It may take some time for the change to apply. Check aka.ms/homesites for details."
+    ```
+
+=== "Text"
+
+    ```text
+    The Home site has been set to https://contoso.sharepoint.com and the Viva Connections default experience to True. It may take some time for the change to apply. Check aka.ms/homesites for details.
+    ```
+
+=== "CSV"
+
+    ```csv
+    The Home site has been set to https://contoso.sharepoint.com and the Viva Connections default experience to True. It may take some time for the change to apply. Check aka.ms/homesites for details.
+    ```
+
 
 === "Markdown"
```

Comment:
@milanholemans : Is the default response the one without the parameter 'vivaConnectionsDefaultStart'?

Reply:
Yes the default response is the one without `vivaConnectionsDefaultStart`. And then we add another response with `vivaConnectionsDefaultStart` set to true because people will get another response then.